### PR TITLE
feat(signature-collection): Sign parliamentary collection

### PR DESCRIPTION
**Pull request by albinagu**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
	- Added support for the "Parliamentary List Signing" application type, enhancing template loading capabilities.
	- Introduced a user-friendly completion interface and structured forms for the signing process.
	- Implemented internationalized error messages and application messages for improved user guidance.
	- Added phone number formatting and state lifecycle management functionalities.

- **Documentation**
	- New README file detailing library usage and testing instructions.

- **Configuration**
	- Introduced TypeScript configuration files tailored for library and testing purposes.

- **Error Handling**
	- Defined internationalized error messages for various application validation scenarios.

### CODE REVIEW

**Review by baering — APPROVED** [review]
Core file LGTM

**Review by Toti91 — APPROVED** [review]
LGTM

File: libs/application/templates/signature-collection/parliamentary-list-signing/src/lib/errors.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+export const errorMessages = {
+  age: defineMessages({
+    title: {
+      id: 'slc.application:error.age.title',
+      defaultMessage: 'Aldur uppfyllir ekki skilyrði',
+      description: '',
+    },
+    summary: {
+      id: 'slc.application:error.age.summary#markdown',
+      defaultMessage: '35 ára aldursmarki er ekki náð',
```
Comment (juni-haukur) [inline libs/application/templates/signature-collection/parliamentary-list-signing/src/lib/errors.ts:12]: nit: forsetaleifar

File: libs/application/templates/signature-collection/parliamentary-list-signing/src/lib/errors.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  }),
+  citizenship: defineMessages({
+    title: {
+      id: 'slc.application:error.citizenship.title',
+      defaultMessage: 'Ekki með íslenkst ríkisfang',
+      description: '',
+    },
+    summary: {
+      id: 'slc.application:error.citizenship.summary#markdown',
+      defaultMessage: 'Þú þarft að vera með íslenkst ríkisfang',
```
Comment (juni-haukur) [inline libs/application/templates/signature-collection/parliamentary-list-signing/src/lib/errors.ts:24]:
```suggestion
      defaultMessage: 'Þú þarft að vera með íslenskt ríkisfang',
```

**Review by juni-haukur — APPROVED** [review]

**Pull request merged**